packages/mwp-app-render/src/components/Dom.jsx

```diff
@@ -57,6 +57,24 @@ const DOM = props => {
 		escapedState,
 	};
 
+	// simple-universal-style-loader accumulates CSS Module style rules
+	// in node's `global` object under the `__styles__` key.
+	//
+	// If this key is populated, we render a single style tag with all
+	// module styles needed for the request.
+	let moduleStyleTag;
+	if ( global.__styles__ ) {
```

Comment:
because referencing `global` is dirty, let's use `getStyles` https://github.com/janoist1/simple-universal-style-loader#server-environment